test/markup/swift/functions.expect.txt

```diff
@@ -5,6 +5,6 @@
 
 <span class="hljs-class"><span class="hljs-keyword">class</span> <span class="hljs-title">MyClass</span> </span>{
   <span class="hljs-function"><span class="hljs-keyword">func</span> <span class="hljs-title">f</span><span class="hljs-params">()</span></span> {
-    <span class="hljs-keyword">return</span> <span class="hljs-literal">true</span>
+    <span class="hljs-keyword">return</span> <span class="hljs-keyword">true</span>
```

Comment:
**This should not have changed.**

Reply:
Yeah, Swift lists `true`, `false`, and `nil` as both keywords and literals, and Xcode highlights them as keyword, which is why I've included them here.
But I'll change them back to the `literal` class.

Reply:
Done.